### ISSUE TITLE
Make a new release?

### ISSUE THREAD

**Issue by tseemann** [hyattpd/Prodigal#41]
Some bugs have been fixed, a new release would be good?

**Comment by smdabdoub** [comment]
> Some bugs have been fixed

Do you mean the pending pull request? I don't see any updates to the main repo.

**Comment by tseemann** [comment]
@smdabdoub there has been many changes since the 2.6.1 release on Aug 21, 2014 on the GoogleImport branch which I think is still the default for 2.x

https://github.com/hyattpd/Prodigal/commits/GoogleImport

**Comment by sjackman** [comment]
The only commit in the default branch that's not included in the release of 2.6.3 is https://github.com/hyattpd/Prodigal/issues/34 `Undefined behavior leads to different program outputs when compiled with GCC 4-6 vs GCC 7`

**Comment by hwalinga** [comment]
I can reproduce #34 with gcc 8.2.0-2 (Currently in Debian testing). So, that problem will be more widespread in the long run (when this enters Ubuntu and so on). Because #34 makes the output have an invalid GFF3 format, it also broke a pipeline I ran (CrisprCasFinder).

Installed with linux brew.  

EDIT:

Since some might find this helpfull and, also since coincedentally a linuxbrew maintainer and writer of the Prodigal formula is in this thread (hi @sjackman ). Here is how to install the latest and fixed version of prodigal using linuxbrew. 

First, create a head reference in the brew formula: Edit this formula with `brew edit prodigal`, and add the following line after the class definition (and maybe this formula will also be updated accordingly by @sjackman ).

```
class Prodigal < Formula                                                        
  head "https://github.com/hyattpd/Prodigal.git", :branch => "GoogleImport" 
  ...
```

You can now install from this created head with

```
brew install --HEAD prodigal 
```

**Comment by sjackman** [comment]
Another option is to add…

```ruby
revision 1
…
patch do
  url "https://patch-diff.githubusercontent.com/raw/hyattpd/Prodigal/pull/35.patch?full_index=1"
  sha256 "fd292c0a98412a7f2ed06d86e0e3f96a9ad698f6772990321ad56985323b99a6"
end
```

@hwalinga I'd be happy to merge a PR to `Brewsci/bio` to add that patch.

**Comment by hyattpd** [comment]
Why does #34 produce an invalid GFF3?  That shouldn't happen (did it change the output in some way?)

**Comment by sjackman** [comment]
Renato (@unode) reports that issue #34 was fixed by PR #35.

**Comment by hyattpd** [comment]
At this point, not really planning any new releases of C version unless there arises a serious bug that causes segmentation faults or the like.
